microsoft/surface/firmware/default.nix

```diff
@@ -2,5 +2,7 @@
 {
   hardware.enableAllFirmware = true;
   hardware.firmware = [
+    # TODO: Wrap with an option:
+    (pkgs.callPackage ./ath10k.nix {})
```

Comment:
Maybe just re-use `config.hardware.enableAllFirmware`?

Reply:
> Maybe just re-use config.hardware.enableAllFirmware?

Are you referring to the `TODO` about wrapping with an option?
If so:  because that essentially "breaks" the original QCA6174 firmware drop by deleting the `board.bin` and `board-2.bin` files, and replacing with this new one, it felt appropriate to let people explicitly choose this specific part, rather than force it on them.

Reply:
Fair point.